docs/BR.md

```diff
@@ -1756,153 +1754,1152 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+| \ \ \ \ `issuerUniqueID`       | MUST NOT be present |
+| \ \ \ \ `subjectUniqueID`      | MUST NOT be present |
+| \ \ \ \ `extensions`           | See [Section 7.1.2.7.1](#71271-subscriber-certificate-types) |
+| `signatureAlgorithm`           | Encoded value MUST be byte-for-byte identical to the `tbsCertificate.signature`. |
+| `signature`                    | |
+
+##### 7.1.2.7.1 Subscriber Certificate Types
+
+There are four types of Subscriber Certificates that may be issued, which vary based on the amount of Subject Information that is included.  Each of these certificate types shares a common profile, with three exceptions: the `subject` name fields that may occur, how those fields are validated, and the contents of the `certificatePolicies` extension. 
+
+| __Type__                    | __Description__                                       |
+| ---                         | -------                                                |
+| Domain Validated (DV)       | See [Section 7.1.2.7.2](#71272-domain-validated)       |
+| Individual Validated (IV)   | See [Section 7.1.2.7.3](#71273-individual-validated)   |
+| Organization Validated (OV) | See [Section 7.1.2.7.4](#71274-organization-validated) |
+| Extended Validation (EV)    | See [Section 7.1.2.7.5](#71275-extended-validation)    |
+
+**Note**: Although each Subscriber Certificate type varies in Subject Information, all Certificates provide the same level of assurance of the device identity (domain name and/or IP address).
+
+##### 7.1.2.7.2 Domain Validated
+
+For a Subscriber Certificate to be Domain Validated, it MUST meet the following profile:
+
+| __Field__             | __Requirements__     |
+| --                    | -------              |
+| `subject`             | See following table. |
+| `certificatePolicies` | MUST be present and MUST assert the [Reserved Certificate Policy Identifier](#7161-reserved-certificate-policy-identifiers) of `2.23.140.1.2.1` as a `policyIdentifier`. |
+| All other extensions  | See [Section 7.1.2.7.6](#71276-subscriber-certificate-extensions) |
+
+All `subject` names MUST be encoded as specified in [Section 7.1.4](#714-name-forms).
+
+The following table details the acceptable `AttributeType`s that may appear within the `type` field of an `AttributeTypeAndValue`, as well as the contents permitted within the `value` field.
+
+Table: Domain Validated `subject` Attributes
+
+| __Attribute Name__       | __Presence__ | __Value__   | __Verification__ |
+| ---                      | -            | ------      | -- |
+| `countryName`            | MAY          | The two-letter ISO 3166-1 country code for the country associated with the Subject. | [Section 3.2.2.3](#3223-verification-of-country) |
+| `commonName`             | SHOULD NOT   | If present, MUST contain a single IP address or Fully-Qualified Domain Name that is one of the values contained in the Certificate's `subjectAltName` extension. | |
```

Comment:
Is the commonname in TLS certificates really a SHOULD NOT?  We all include it, especially DV TLS where it's the only field in the SubjectDN. Given the latest _guidance_ on how to interpret SHOULD and SHOULD NOT, what's the best way to remain compliant with the BRs and auditors?  Would it be better to use MAY instead, until such time it's prohibited?

Reply:
> Is the commonname in TLS certificates really a SHOULD NOT?

Yes

> Given the latest guidance on how to interpret SHOULD and SHOULD NOT, what's the best way to remain compliant with the BRs and auditors? 

Can you clarify this remark? Compliance with the BRs here is defined by reference to [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119) , so there shouldn’t be any issues here.

> This phrase, or the phrase "NOT RECOMMENDED" mean that
>   there may exist valid reasons in particular circumstances when the
>   particular behavior is acceptable or even useful, but the full
>   implications should be understood and the case carefully weighed
>   before implementing any behavior described with this label.

Consider this language (emphasis added) from [RFC 2818](https://datatracker.ietf.org/doc/html/rfc2818#section-3.1)

> Althoug the use of the Common Name is existing practice, it is deprecated and
>   **Certification Authorities are encouraged to use the dNSName _instead_.**


Is the substance of the concern that it shouldn’t say SHOULD NOT for things folks presently do? Does that concern still exist if we substitute the RFC 2119 equivalent language, NOT RECOMMENDED?

Reply:
I like NOT RECOMMENDED, yes, that would alleviate my concern.

Reply:
Switched all the language to NOT RECOMMENDED. Even though it's semantically the same as SHOULD NOT, we'll see if this makes it more palatable :)